ldap version 2023.3.0 fails to start

**Describe the bug**
ldap version 2023.3.0 fails to start. Stuck in a boot loop.

**To Reproduce**
Steps to reproduce the behavior:
1. Create an ldap outpost
2. See error in docker logs

**Expected behavior**
Ldap container starts.

**Logs**
```
DBG undefined | event=Loaded config from environment timestamp=2023-03-13T19:16:14Z 
timestamp="2023-03-13T19:16:14Z" level=info event="Starting Debug server" listen="0.0.0.0:9900" logger=authentik.go_debugger
timestamp="2023-03-13T19:16:15Z" level=panic event="Failed to run server" error="500 Internal Server Error"
timestamp="2023-03-13T19:16:15Z" level=info event="finished shutdown" logger=authentik.outpost.ak-api-controller
panic: (*logrus.Entry) 0xc0002140e0

goroutine 1 [running]:
github.com/sirupsen/logrus.(*Entry).log(0xc000214070, 0x0, {0xc00003a0d8, 0x14})
	/go/pkg/mod/github.com/sirupsen/logrus@v1.9.0/entry.go:260 +0x4d6
github.com/sirupsen/logrus.(*Entry).Log(0xc000214070, 0x0, {0xc000623e50?, 0xfc7060?, 0xc0002d20c0?})
	/go/pkg/mod/github.com/sirupsen/logrus@v1.9.0/entry.go:304 +0x4f
github.com/sirupsen/logrus.(*Entry).Panic(...)
	/go/pkg/mod/github.com/sirupsen/logrus@v1.9.0/entry.go:342
main.main()
	/go/src/goauthentik.io/cmd/ldap/main.go:73 +0x585
DBG undefined | event=Loaded config from environment timestamp=2023-03-13T19:16:16Z 
timestamp="2023-03-13T19:16:16Z" level=info event="Starting Debug server" listen="0.0.0.0:9900" logger=authentik.go_debugger
timestamp="2023-03-13T19:16:16Z" level=panic event="Failed to run server" error="500 Internal Server Error"
timestamp="2023-03-13T19:16:16Z" level=info event="finished shutdown" logger=authentik.outpost.ak-api-controller
panic: (*logrus.Entry) 0xc0001a8230

goroutine 1 [running]:
github.com/sirupsen/logrus.(*Entry).log(0xc0001a8150, 0x0, {0xc00003a1c8, 0x14})
	/go/pkg/mod/github.com/sirupsen/logrus@v1.9.0/entry.go:260 +0x4d6
github.com/sirupsen/logrus.(*Entry).Log(0xc0001a8150, 0x0, {0xc0005abe50?, 0xfc7060?, 0xc000132200?})
	/go/pkg/mod/github.com/sirupsen/logrus@v1.9.0/entry.go:304 +0x4f
github.com/sirupsen/logrus.(*Entry).Panic(...)
	/go/pkg/mod/github.com/sirupsen/logrus@v1.9.0/entry.go:342
main.main()
	/go/src/goauthentik.io/cmd/ldap/main.go:73 +0x585
DBG undefined | event=Loaded config from environment timestamp=2023-03-13T19:16:17Z 
timestamp="2023-03-13T19:16:17Z" level=info event="Starting Debug server" listen="0.0.0.0:9900" logger=authentik.go_debugger
timestamp="2023-03-13T19:16:18Z" level=panic event="Failed to run server" error="500 Internal Server Error"
timestamp="2023-03-13T19:16:18Z" level=info event="finished shutdown" logger=authentik.outpost.ak-api-controller
panic: (*logrus.Entry) 0xc0000ec3f0

goroutine 1 [running]:
github.com/sirupsen/logrus.(*Entry).log(0xc0000ec380, 0x0, {0xc00003aa38, 0x14})
	/go/pkg/mod/github.com/sirupsen/logrus@v1.9.0/entry.go:260 +0x4d6
github.com/sirupsen/logrus.(*Entry).Log(0xc0000ec380, 0x0, {0xc00056fe50?, 0xfc7060?, 0xc0004320c0?})
	/go/pkg/mod/github.com/sirupsen/logrus@v1.9.0/entry.go:304 +0x4f
github.com/sirupsen/logrus.(*Entry).Panic(...)
	/go/pkg/mod/github.com/sirupsen/logrus@v1.9.0/entry.go:342
main.main()
	/go/src/goauthentik.io/cmd/ldap/main.go:73 +0x585
DBG undefined | event=Loaded config from environment timestamp=2023-03-13T19:16:19Z 
timestamp="2023-03-13T19:16:19Z" level=info event="Starting Debug server" listen="0.0.0.0:9900" logger=authentik.go_debugger
timestamp="2023-03-13T19:16:19Z" level=panic event="Failed to run server" error="500 Internal Server Error"
timestamp="2023-03-13T19:16:19Z" level=info event="finished shutdown" logger=authentik.outpost.ak-api-controller
panic: (*logrus.Entry) 0xc00055a930

goroutine 1 [running]:
github.com/sirupsen/logrus.(*Entry).log(0xc00055a8c0, 0x0, {0xc00003a618, 0x14})
	/go/pkg/mod/github.com/sirupsen/logrus@v1.9.0/entry.go:260 +0x4d6
github.com/sirupsen/logrus.(*Entry).Log(0xc00055a8c0, 0x0, {0xc000509e50?, 0xfc7060?, 0xc0001324c0?})
	/go/pkg/mod/github.com/sirupsen/logrus@v1.9.0/entry.go:304 +0x4f
github.com/sirupsen/logrus.(*Entry).Panic(...)
	/go/pkg/mod/github.com/sirupsen/logrus@v1.9.0/entry.go:342
main.main()
	/go/src/goauthentik.io/cmd/ldap/main.go:73 +0x585
DBG undefined | event=Loaded config from environment timestamp=2023-03-13T19:16:20Z 
timestamp="2023-03-13T19:16:20Z" level=info event="Starting Debug server" listen="0.0.0.0:9900" logger=authentik.go_debugger
timestamp="2023-03-13T19:16:21Z" level=panic event="Failed to run server" error="500 Internal Server Error"
timestamp="2023-03-13T19:16:21Z" level=info event="finished shutdown" logger=authentik.outpost.ak-api-controller
panic: (*logrus.Entry) 0xc0001d2310

goroutine 1 [running]:
github.com/sirupsen/logrus.(*Entry).log(0xc0001d22a0, 0x0, {0xc0003d8000, 0x14})
	/go/pkg/mod/github.com/sirupsen/logrus@v1.9.0/entry.go:260 +0x4d6
github.com/sirupsen/logrus.(*Entry).Log(0xc0001d22a0, 0x0, {0xc000743e50?, 0xfc7060?, 0xc000300080?})
	/go/pkg/mod/github.com/sirupsen/logrus@v1.9.0/entry.go:304 +0x4f
github.com/sirupsen/logrus.(*Entry).Panic(...)
	/go/pkg/mod/github.com/sirupsen/logrus@v1.9.0/entry.go:342
main.main()
	/go/src/goauthentik.io/cmd/ldap/main.go:73 +0x585


```

**Version and Deployment (please complete the following information):**
 - authentik version: 2023.3.0
 - Deployment: docker

**Additional context**
Add any other context about the problem here.

--- a/authentik/providers/ldap/api.py
+++ b/authentik/providers/ldap/api.py
@@ -26,6 +26,7 @@ class LDAPProviderSerializer(ProviderSerializer):
             "search_mode",
             "bind_mode",
         ]
+        extra_kwargs = ProviderSerializer.Meta.extra_kwargs
 
 
 class LDAPProviderViewSet(UsedByMixin, ModelViewSet):

--- a/authentik/providers/oauth2/api/providers.py
+++ b/authentik/providers/oauth2/api/providers.py
@@ -39,6 +39,7 @@ class OAuth2ProviderSerializer(ProviderSerializer):
             "issuer_mode",
             "jwks_sources",
         ]
+        extra_kwargs = ProviderSerializer.Meta.extra_kwargs
 
 
 class OAuth2ProviderSetupURLs(PassiveSerializer):

--- a/authentik/providers/proxy/api.py
+++ b/authentik/providers/proxy/api.py
@@ -95,6 +95,7 @@ class ProxyProviderSerializer(ProviderSerializer):
             "refresh_token_validity",
             "outpost_set",
         ]
+        extra_kwargs = ProviderSerializer.Meta.extra_kwargs
 
 
 class ProxyProviderViewSet(UsedByMixin, ModelViewSet):

--- a/authentik/providers/saml/api/providers.py
+++ b/authentik/providers/saml/api/providers.py
@@ -154,6 +154,7 @@ class SAMLProviderSerializer(ProviderSerializer):
             "url_slo_post",
             "url_slo_redirect",
         ]
+        extra_kwargs = ProviderSerializer.Meta.extra_kwargs
 
 
 class SAMLMetadataSerializer(PassiveSerializer):

--- a/schema.yml
+++ b/schema.yml
@@ -30195,7 +30195,6 @@ components:
         authorization_flow:
           type: string
           format: uuid
-          nullable: true
           description: Flow used when authorizing this provider.
         property_mappings:
           type: array
@@ -30268,6 +30267,7 @@ components:
       required:
       - assigned_application_name
       - assigned_application_slug
+      - authorization_flow
       - component
       - meta_model_name
       - name
@@ -30285,7 +30285,6 @@ components:
         authorization_flow:
           type: string
           format: uuid
-          nullable: true
           description: Flow used when authorizing this provider.
         property_mappings:
           type: array
@@ -30328,6 +30327,7 @@ components:
         bind_mode:
           $ref: '#/components/schemas/LDAPAPIAccessMode'
       required:
+      - authorization_flow
       - name
     LDAPSource:
       type: object
@@ -30927,7 +30927,6 @@ components:
         authorization_flow:
           type: string
           format: uuid
-          nullable: true
           description: Flow used when authorizing this provider.
         property_mappings:
           type: array
@@ -31027,6 +31026,7 @@ components:
       required:
       - assigned_application_name
       - assigned_application_slug
+      - authorization_flow
       - component
       - meta_model_name
       - name
@@ -31043,7 +31043,6 @@ components:
         authorization_flow:
           type: string
           format: uuid
-          nullable: true
           description: Flow used when authorizing this provider.
         property_mappings:
           type: array
@@ -31121,6 +31120,7 @@ components:
               authenticate.
           title: Any JWT signed by the JWK of the selected source can be used to authenticate.
       required:
+      - authorization_flow
       - name
     OAuth2ProviderSetupURLs:
       type: object
@@ -35608,7 +35608,6 @@ components:
         authorization_flow:
           type: string
           format: uuid
-          nullable: true
           description: Flow used when authorizing this provider.
         property_mappings:
           type: array
@@ -35838,7 +35837,6 @@ components:
         authorization_flow:
           type: string
           format: uuid
-          nullable: true
           description: Flow used when authorizing this provider.
         property_mappings:
           type: array
@@ -36299,7 +36297,6 @@ components:
         authorization_flow:
           type: string
           format: uuid
-          nullable: true
           description: Flow used when authorizing this provider.
         property_mappings:
           type: array
@@ -36426,7 +36423,6 @@ components:
         authorization_flow:
           type: string
           format: uuid
-          nullable: true
           description: Flow used when authorizing this provider.
         property_mappings:
           type: array
@@ -37873,7 +37869,6 @@ components:
         authorization_flow:
           type: string
           format: uuid
-          nullable: true
           description: Flow used when authorizing this provider.
         property_mappings:
           type: array
@@ -37981,6 +37976,7 @@ components:
       required:
       - assigned_application_name
       - assigned_application_slug
+      - authorization_flow
       - client_id
       - component
       - external_host
@@ -38001,7 +37997,6 @@ components:
         authorization_flow:
           type: string
           format: uuid
-          nullable: true
           description: Flow used when authorizing this provider.
         property_mappings:
           type: array
@@ -38075,6 +38070,7 @@ components:
           description: 'Tokens not valid on or after current time + this value (Format:
             hours=1;minutes=2;seconds=3).'
       required:
+      - authorization_flow
       - external_host
       - name
     RedirectChallenge:
@@ -38315,7 +38311,6 @@ components:
         authorization_flow:
           type: string
           format: uuid
-          nullable: true
           description: Flow used when authorizing this provider.
         property_mappings:
           type: array
@@ -38431,6 +38426,7 @@ components:
       - acs_url
       - assigned_application_name
       - assigned_application_slug
+      - authorization_flow
       - component
       - meta_model_name
       - name
@@ -38471,7 +38467,6 @@ components:
         authorization_flow:
           type: string
           format: uuid
-          nullable: true
           description: Flow used when authorizing this provider.
         property_mappings:
           type: array
@@ -38542,6 +38537,7 @@ components:
             * `post` - Post
       required:
       - acs_url
+      - authorization_flow
       - name
     SAMLSource:
       type: object

